src/components/table/table-column-resize/resizable-table-column.component.ts

```diff
@@ -21,6 +21,11 @@ export class ResizableTableColumnComponent implements AfterViewInit, OnDestroy {
     @HostBinding('class.ux-resizable-table-column-disabled')
     disabled: boolean = false;
 
+    /** Show/Hide column resizable handle */
+    @Input()
+    @HostBinding('class.ux-resizable-table-hide-handle')
+    hideHandle: boolean = false;
```

Comment:
Check this item on the tips & tricks page, we should be coercing booleans in new properties:

https://github.houston.softwaregrp.net/caf/ux-aspects-team/wiki/Tips-and-Tricks#coerce-inputs